### DISCUSSION TITLE
The Chinese document has been completed, see where it is appropriate to distribute it.

### DISCUSSION THREAD

**Discussion by lecepin** [pmndrs/jotai#1690] [General]
It took a long time to finally translate Jotai's documentation into Chinese.

Github: https://github.com/lecepin/jotai-docs-cn

Hi @dai-shi,
Can this document link be placed in the official document, or other suitable places, for some people to read?

**Comment by dai-shi** [comment]
We are discussing internally about the website and it will take time because we have something to work with priority.
Meanwhile, we have links in jotai-labs org and that's the suitable place.
Would you send a PR to https://github.com/jotai-labs/links ?

**Reply by lecepin** [reply]
All right!
